docs/modules/servers/partials/ClamAVScan.adoc

```diff
@@ -30,6 +30,9 @@ The default is *6*.
 * *<pingIntervalMilli>*: the interval (in milliseconds)
```

Comment:
Lets use `DurationParser` here too please.

We can default to MilliSecond when no unit in order to preserve inter-operability.